Testing/xASL_test_SPM.m

```diff
@@ -0,0 +1,126 @@
+function xASL_test_SPM(TestDirDest, bTestDataUsed)
```

Comment:
I'm confused. This function used to be inside xASL_test_TestExploreASL. I'm fine having it independently, but then we should remove it from the other function to avoid having two versions.

Reply:
I'll remove the one in xASL_test_TestExploreASL.